web/client/product/pages/Maps.jsx

```diff
@@ -7,7 +7,7 @@
  */
 const React = require('react');
 
-require('../assets/css/viewer.css');
+// require('../assets/css/viewer.css');
```

Comment:
Why did you removed it, there are still some classes specific for the product viewer, isn't it?